pkg/controller/volume/persistentvolume/scheduler_binder_cache.go

```diff
@@ -59,6 +60,8 @@ func (c *podBindingCache) DeleteBindings(pod *v1.Pod) {
 
 	podName := getPodName(pod)
 	delete(c.bindings, podName)
+
+	metrics.VolumeBindingDeleteFromSchedulerBinderCache.Inc()
```

Comment:
I think it would be better to only record when it's actually removed (it could have already been removed previously)